src/pages/home/report/ReportActionItemSingle.js

```diff
@@ -44,7 +49,7 @@ const ReportActionItemSingle = ({
     // we should stop referring to the report history items entirely for this information.
     const personArray = displayName ? [{type: 'TEXT', text: displayName}] : action.person;
     return (
-        <View style={[styles.chatItem]}>
+        <View style={outerViewStyles}>
```

Comment:
Exposing the style choice to the parent seemed the simplest solution here. Perhaps it would be better to add a prop to change the specific change, but I think this could get overly complex.